cern.root.ROOT.yaml

```diff
@@ -0,0 +1,26 @@
+app-id: cern.root.ROOT
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: root
+rename-desktop-file: root
+rename-icon: Root6Icon #128x128
+finish-args:
+  - --socket=fallback-x11
```

Comment:
`--share=ipc` is necessary

Reply:
this is not the question.

Reply:
Sorry, but I thought x11 needed ipc, that's why I put it.

Reply:
This is exactly what I'm saying

![image](https://github.com/flathub/flathub/assets/114441/7142b3ed-2fb0-4684-8632-b3cb51ae8551)

Reply:
this wasn't resolved while you marked it as such.